Create ui-c8y-1023-87-0-ux-revisited-widget-configurations-sections-preview-and-screenshots-backport-release-cd.md (#4814)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-87-0-ux-revisited-widget-configurations-sections-preview-and-screenshots-backport-release-cd.md

---
date: ""
title: Reorganized widget configuration view with a preview section
product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66390
version: 1023.87.0
---

> The widget configuration view has been reorganized for better consistency and now includes a preview section that shows how settings will appear before they are applied. Also updated the screenshots of available widgets in the widget selection.

Reviewer: @carlosceia Does the second sentence refer to the Codex?

Reviewer 2: no, it's actually for the widget selection drawer (it's displayed when you click "add widget" in a dashboard)

> The widget configuration view has been reorganized for better consistency and now includes a preview section that shows how settings will appear before they are applied. Also updated the screenshots of available widgets in the widget selection.

Reviewer: ```suggestion
The widget configuration view has been reorganized for better consistency and now includes a preview section that shows how settings will appear before they are applied. Moreover, the screenshots of the available widgets in the widget selection drawer have been updated.
```

---
date: ""

> title: ux revisited widget configurations sections preview and screenshots [BACKPORT] [release/cd] (#12217)

Reviewer 2: ```suggestion
title: Reorganized widget configuration view with a preview section
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66390
version: 1023.87.0
---
Backport of Carlos PR ->
https://github.com/Cumulocity-IoT/cumulocity-ui/pull/11769

> Also fixes: https://cumulocity.atlassian.net/browse/MTM-66579

Reviewer 2: ```suggestion
The widget configuration view has been reorganized for better consistency and now includes a preview section that shows how settings will appear before they are applied. Also updated the screenshots of available widgets in the widget selection.
```
